api/Element.json

```diff
@@ -2627,9 +2627,10 @@
             "deprecated": false
           }
         },
-        "init_clonable_parameter": {
+        "options_clonable_parameter": {
```

Comment:
Note, changed these to options, because that is what the MDN docs use. I'd be OK with since these are "initialize the shadow root" but options is more common.

Reply:
Yeah options is more common. I suspect this comes from the IDL which uses init rather than options. 🤷